custom/plugins/emspay/Resources/config.xml

```diff
@@ -56,12 +64,16 @@
             <description lang="fr">Vous pouvez spécifier des adresses IP spécifiques pour lesquelles AfterPay est visible, par exemple si vous souhaitez tester AfterPay. Vous pouvez taper des adresses IP comme 128.0.0.1, 255.255.255.255. Si vous laissez ce champ AfterPay vide, il est visible par toutes les adresses IP.</description>
         </element>
         <element type="text" scope="locale">
-            <name>emsonline_test_ip_klarna</name>
-            <label>EMS Online Klarna Pay Later Test IP</label>
-            <description lang="de">Sie können bestimmte IP-Adressen angeben, für die Klarna Pay Later sichtbar ist, z. B. wenn Sie Klarna Pay Later testen möchten. Sie können IP-Adressen als 128.0.0.1, 255.255.255.255 eingeben. Wenn Sie dieses Feld leer lassen, ist Klarna Pay Later für alle IP-Adressen sichtbar. "</description>
-            <description lang="en">You can specify specific IP addresses for which the Klarna is used Payment later payment method is visible, eg if you want to test (for example: 128.0.0.1, 255.255.255.255). If you do not enter anything, the payment method is visible for all IP addresses.</description>
-            <description lang="nl">U kunt specifieke IP-adressen specificeren waarvoor Klarna Pay Later zichtbaar is, bijvoorbeeld als u Klarna Pay Later wilt testen. U kunt IP-adressen typen als 128.0.0.1, 255.255.255.255. Als u dit leeg laat, is Klarna Pay Later zichtbaar voor alle IP-adressen. "</description>
-            <description lang="fr">Vous pouvez spécifier des adresses IP spécifiques pour lesquelles Klarna Pay Later est visible, par exemple si vous souhaitez tester Klarna Pay Later. Vous pouvez taper des adresses IP comme 128.0.0.1, 255.255.255.255. Si vous laissez ce Klarna Pay Later vide, il est visible par toutes les adresses IP. "</description>
+            <name>emsonline_afterpay_countries</name>
+            <label>Countries available for AfterPay</label>
+            <label lang="de">Für AfterPay verfügbare Länder</label>
+            <label lang="nl">Landen beschikbaar voor AfterPay</label>
+            <label lang="fr">Pays disponibles pour AfterPay</label>
+            <value>NL, BE</value>
+            <description lang="en">To allow AfterPay to be used for any other country just add its country code (in ISO 2 standard) to the "Countries available for AfterPay" field. Example: BE, NL, FR.</description>

```

Comment:
Needs add description for empty values:
EN "<br> If field is empty then AfterPay will be available for all countries."
DE "<br> Wenn das Feld leer ist, ist AfterPay fur alle Lander verfugbar."
FR "<br> Si le champ est vide, AfterPay sera disponible pour tous les pays."
NL "<br> Als het veld leeg is, is AfterPay beschikbaar voor alle landen."

Reply:
Thanks for localized answer,